code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -135,7 +135,7 @@
 	category = "Defensive"
 
 /datum/spellbook_entry/lightningPacket
-	name = "Lightning bolt!  Lightning bolt!"

```

Comment:
Aww, I thought the joke was funny.

https://www.youtube.com/watch?v=j_ekugPKqFw